File: 2017.1/2017-04-25/Services/app/src/main/java/br/ufpe/cin/if1001/services/MainActivity.java
package br.ufpe.cin.if1001.services;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final Button btn_NoBindingService = (Button) findViewById(R.id.botao_musicServiceWithoutBinding);
        btn_NoBindingService.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                startActivity(new Intent(getApplicationContext(),MusicPlayerNoBindingActivity.class));
            }
        });

        final Button btn_BindingService  = (Button) findViewById(R.id.botao_musicServiceWithBinding);
        btn_BindingService.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                startActivity(new Intent(getApplicationContext(),MusicPlayerWithBindingActivity.class));
            }
        });

        final Button btn_intentService = (Button) findViewById(R.id.botao_downloadService);
        btn_intentService.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                startActivity(new Intent(getApplicationContext(),DownloadActivity.class));
            }
        });

        final Button btn_broadcast = (Button) findViewById(R.id.botao_enviaBroadcast);
        btn_broadcast.setOnClickListener(new View.OnClickListener() {
            public void onClick(View src) {
                sendBroadcast(new Intent("br.ufpe.cin.if1001.broadcasts.exemplo"));
            }
        });
    }
}

File: 2019-04-03/DataManagement/app/src/main/java/br/ufpe/cin/android/datamanagement/MainActivity.kt
package br.ufpe.cin.android.datamanagement

import androidx.appcompat.app.AppCompatActivity
import android.content.Intent
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnMemoriaExterna.setOnClickListener {
            startActivity(Intent(applicationContext,MemoriaExternaActivity::class.java))
        }
        btnMemoriaInterna.setOnClickListener {
            startActivity(Intent(applicationContext,MemoriaInternaActivity::class.java))
        }
        btnPreferencesActivity.setOnClickListener {
            startActivity(Intent(applicationContext,PrefsActivity::class.java))
        }
        btnPreferencesMenuActivity.setOnClickListener {
            startActivity(Intent(applicationContext,PrefsMenuActivity::class.java))
        }
        btnSharedPreferences.setOnClickListener {
            startActivity(Intent(applicationContext,SharedPrefsActivity::class.java))
        }
        btnSQLite.setOnClickListener {
            startActivity(Intent(applicationContext,SQLiteActivity::class.java))
        }
        btnRoom.setOnClickListener {
            startActivity(Intent(applicationContext,RoomActivity::class.java))
        }
    }
}
